vault/resource_namespace.go

```diff
@@ -64,6 +64,9 @@ func namespaceRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*api.Client)
 
 	path := d.Get("path").(string)
+	if path == "" {
```

Comment:
Since the `"path"` field is marked as required above, is it possible to hit this case?